next/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "next",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "14.2.4",
+    "react": "^18",
+    "react-dom": "^18"
+  },
+  "devDependencies": {
```

Comment:
Can we add typescript in dev deps here too so that we can avoid any version conflict or compatibility issues?

Reply:
We are not using typescript in this project

Reply:
@ankur-dwivedi `next-env.d.ts` is still there.